### PR TITLE
Add windows support for Scatter/Gather I/O Test

### PR DESCRIPTION
**Pull request by kimi1978wy**
Get package info with Offload.TxLSO enabled and disabled

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1479119

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(1/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.enable_scatter_windows: PASS (227.78 s)
 (2/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.enable_scatter_windows: PASS (246.74 s)
 (3/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.enable_scatter_windows: PASS (183.04 s)
 (4/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.enable_scatter_windows: PASS (216.73 s)
 (5/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.enable_scatter_windows: PASS (207.40 s)
 (6/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.enable_scatter_windows: PASS (198.08 s)
 (7/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.enable_scatter_windows: PASS (219.82 s)
(8/8) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.enable_scatter_windows: PASS (534.06 s)

**Comment by yanglei-rh** [comment]
(2/2) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (189.14 s)

**Comment by kimi1978wy** [comment]
Test on RHEL8 host:
(01/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win10.i386.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (345.78 s)
 (02/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (352.43 s)
 (03/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (264.95 s)
 (04/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (313.49 s)
 (05/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (270.08 s)
 (06/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (285.52 s)
 (07/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (304.07 s)
 (08/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (288.68 s)
 (09/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (279.59 s)
 (10/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (270.90 s)

Test on RHEL7 host:
 (1/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (379.36 s)
 (2/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (291.07 s)
 (3/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (283.07 s)
 (4/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (285.76 s)
 (5/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (313.12 s)
 (6/6) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (291.35 s)

**Comment by yanglei-rh** [comment]
LGTM and Test pass

(1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS
 (3/4) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (206.50 s)
 (4/4) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (175.73 s)
(2/2) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35:
Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (195.87 s)
(3/4) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (184.35 s)
 (4/4) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (181.68 s)

(3/4) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (224.91 s)
 (4/4) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (210.24 s)
(2/2) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.enable_scatter_windows.q35: PASS  (1/1) Host_RHEL.m7.u8.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.enable_scatter_windows.q35: PASS (375.78 s)

Acked By:  Lei Yang leiyang@redhat.com

File: qemu/tests/enable_scatter_windows.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+       some packets length should over 1514
+    5) disable TxLSO and set MTU to 1000
+    6) start file transfer and log file transfer traffic again,
+       no packet length should over 1014
+
+    param test: the test object
+    param params: the test params
+    param env: test environment
+    """
+    def _process_finish(session, process_name):
```
Comment (lolyu) [inline qemu/tests/enable_scatter_windows.py]: better to rename it like `_is_process_finished`

File: qemu/tests/enable_scatter_windows.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        Stop the running wireshark session
+        """
+        error_context.context("Stop wireshark", logging.info)
+        status, output = session.cmd_status_output(stop_wireshark_cmd,
+                                                   timeout=timeout)
+        if status:
+            test.error("Failed to stop wireshark: status=%s, output=%s"
+                       % (status, output))
+
+    def _parse_log_file(filter):
```
Comment (lolyu) [inline qemu/tests/enable_scatter_windows.py]: `filter` is a python builtin, better to rename it.

**Comment by kimi1978wy** [comment]
@lolyu modified, pls review it again

**Pull request merged**